controllers/agents.js

```diff
@@ -550,6 +550,38 @@ router.get('/:agent_id/upgrade_result', function(req, res) {
 })
 
 
+/**
+ * @api {get} /agents/:agent_id/config/:component/:configuration Get loaded configuration from agent
+ * @apiName GetConfig
+ * @apiGroup Config
+ *
+ * @apiParam {Number} agent_id Agent ID.
+ * @apiParam {String} component Selected component.
+ * @apiParam {String} configuration Configuration to read.
+ *
+ * @apiDescription Returns the loaded configuration from agent in JSON format.
+ *
+ * @apiExample {curl} Example usage:
+ *     curl -u foo:bar -k -X GET "https://127.0.0.1:55000/agents/001/config/logcollector/localfile?pretty"
+ *
+ */
+ router.get('/:agent_id/config/:component/:configuration', function(req, res) {
+     logger.debug(req.connection.remoteAddress + " GET /agents/:agent_id/config/:component/:configuration");
+
+     var data_request = {'function': '/agents/:agent_id/config/:component/:configuration', 'arguments': {}};
+     var filters = {'agent_id': 'names', 'component': 'names', 'configuration': 'names'};
```

Comment:
The `agent_id` must be filtered as `numbers`, not as `names`.

https://github.com/wazuh/wazuh-api/blob/03cccb38cab6acebd5feeec91b1db18a5879757a/controllers/agents.js#L508

Reply:
Fixed! 
https://github.com/wazuh/wazuh-api/pull/72/commits/1f663f40ab421f67fb8e7ab86c0d4b7489f0afbb